src/utils/Context.cpp

```diff
@@ -29,37 +29,34 @@ using namespace utils;
 /**
  *
  */
-Context::Context()
+Context::Context(sf::RenderWindow* commonWindow)
 {
+    window = commonWindow;
```

Comment:
check if (sf::RenderWindow* window) and delete line 34 is not enough... maybe it is...